drivers/gpu/drm/vc4/vc4_hdmi.c

```diff
@@ -917,6 +917,45 @@ static void vc4_hdmi_audio_shutdown(struct snd_pcm_substream *substream,
 	vc4_hdmi->audio.substream = NULL;
 }
 
+static int sample_rate_to_mai_fmt(int samplerate)
+{
+   switch(samplerate)
+   {
+      case 8000:
+         return 1;
```

Comment:
It would be good to replace these magic MAI_FMT numbers here (and below) with proper defines in vc4_regs.h